.drone.star

```diff
@@ -36,6 +36,20 @@ dir = {
     "base": "/drone",
 }
 
+gui_tests1 = [
+    "tst_sharing",
+]
+gui_tests2 = [
+    "tst_addAccount",
+    "tst_syncing",
+    "tst_checkAlltabs",
+    "tst_vfs",
+    "tst_deletFilesFolders",
+    "tst_editFiles",
+    "tst_loginLogout",
+    "tst_removeAccountConnection",
+]
+
```

Comment:
Maybe we can use `config` like in other repos:
```
config = {
  gui_tests_1: [
    "tst_sharing",
  ], 
  gui_tests_1: [
    "tst_addAccount",
    ...
  ]
}
```

and refactor `gui_tests` to build pipelines as per the config